packages/governance/src/contractGovernance/paramManager.js

```diff
@@ -102,6 +101,7 @@ const makeParamManagerBuilder = zoe => {
     // to our caller. They must handle them carefully to ensure that they end up
     // in appropriate hands.
     setters[`update${name}`] = setParamValue;
+    setters[`prepareToUpdate${name}`] = () => {};
```

Comment:
void fn:
```suggestion
    setters[`prepareToUpdate${name}`] = () => undefined;
```
do we really need this no-op though? It looks like only the async types need it.

Reply:
> do we really need this no-op though? It looks like only the async types need it.

In updateParams, we're treating the async and sync types polymorphically, so they both need `prepareToUpdate()`. Is there a better way to give the sync types a no-op function?